.docs/user-guide/storage-providers.md

```diff
@@ -675,22 +706,58 @@ The DigitalOcean block storage driver has the following requirements:
 * Valid DigitalOcean [access token](https://goo.gl/iKoAec)
 
 #### Configuration
-The following example illustrates a minimal configuration for the DigitalOcean
+The following is an example with all possible fields configured. For a running
+example see the `Examples` section.
```

Comment:
Agreed. I'll take that enhancement on in a separate doc PR. The majority of the providers have this line [verbatim](http://libstorage.readthedocs.io/en/stable/user-guide/storage-providers/#configuration_8), and none of them have a link.

It's always just the case of taking precedent from an existing driver, and using that as a guide for the next one. :)